docs/assets/js/search.js

```diff
@@ -9,43 +9,56 @@ const searchResultsElm = document.getElementById( 'search-results' );
  * Update page markup with search results.
  * @param {Array} results - A list of search result hits as objects.
  * @param {Object} store - search index/meta data store in the window object.
+ * @returns {boolean} True if there are search results, false otherwise.
  */
 function displaySearchResults( results, store ) {
   // Are there any results?
-  if ( results.length ) {
-    let appendString = '';
-
-    // Iterate over the results.
-    for ( let i = 0; i < results.length; i++ ) {
-      const item = store[results[i].ref];
-      appendString += '<li><a href="../' + item.url + '"><h3>' + item.title + '</h3></a>';
-
-      // Show some preview text under each search results item.
-      let previewText = '';
-      const searchMatchWordFragment = Object.keys( results[i].matchData.metadata )[0];
-      const searchMatchFields = results[i].matchData.metadata[searchMatchWordFragment];
-
-      // Remove fields that should never appear as the preview.
-      delete searchMatchFields.id;
-      delete searchMatchFields.title;
+  if ( results.length === 0 ) {
+    displayNoSearchResults( searchResultsElm, results.searchTerm );
+    return false;
+  }
```

Comment:
```suggestion
```

Because `results` is known before we invoke `displaySearchResults`, I'd prefer to avoid calling the function altogether instead of returning early. See below.

Reply:
Refactored in https://github.com/cfpb/design-system/pull/479/commits/04489eac20472c1a1c293054c09e98dcee1c100b